app/src/main/java/com/hedvig/app/feature/embark/EmbarkViewModel.kt

```diff
@@ -68,10 +70,18 @@ abstract class EmbarkViewModel(
         valueStore.put(key, value)
     }
 
+    fun putInStore(key: String, value: List<String>) {
+        valueStore.put(key, value)
+    }
+
     fun getPrefillFromStore(key: String) = valueStore.prefill.get(key)
 
     fun getFromStore(key: String) = valueStore.get(key)
 
+    fun getQuoteIds(): List<String> = valueStore.getList("quoteIds")
```

Comment:
Can we get around hard coding these keys?

Reply:
Yes! The `Redirect`-component should specify from where in the store to load parameters, hard-coding just won't do 😞

Reply:
Rewrote the `getListFromStore` function in the viewmodel like this: 
```
    fun getListFromStore(keys: List<String>): List<String> {
        return keys.map {
            valueStore.getList(it) ?: listOfNotNull(valueStore.get(it))
        }.flatten()
    }
```